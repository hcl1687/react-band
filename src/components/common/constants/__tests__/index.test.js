import constFactory from '../index.entry'

const CONSTS = constFactory()

describe('common/constants', () => {
  it('should render correctly', () => {
    expect(CONSTS.LOCALE).toMatchSnapshot()
  })
})
