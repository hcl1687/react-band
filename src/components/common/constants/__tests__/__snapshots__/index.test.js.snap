// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/constants should render correctly 1`] = `
Object {
  "en-GB": "en-GB",
  "en-US": "en-US",
  "es-MX": "es-MX",
  "id": "id",
  "zh-CN": "zh-CN",
}
`;
